demo/prototype-deps.spec

```diff
@@ -0,0 +1,26 @@
+Name:       prototype-deps
+Version:    0.1
+Release:    1%{?dist}
+Summary:    This rpm isn't for real usage and meant to contain only dependencies
+URL:        https://github.com/leapp-to/prototype
+License:    SeeProject
+
+BuildRequires: ansible
+BuildRequires: gcc
+BuildRequires: libguestfs-tools-c
+BuildRequires: libvirt-client
+BuildRequires: libvirt-devel
+BuildRequires: nmap
+BuildRequires: pipsi
+BuildRequires: python2-devel
+BuildRequires: redhat-rpm-config
+BuildRequires: rubygem-hitimes
+BuildRequires: rubygem-nio4r
+BuildRequires: vagrant-libvirt
```

Comment:
To make this useful for the CI bootstrap script we need the following packages as well:

- epel-release
- openssl-devel
- python2-pip

Would be good to also make the `centos-ci/bootstrap.sh` use this new file.